src/app.ts

```
@@ -0,0 +1,28 @@
+import express from "express";
+import type { Request, Response, NextFunction } from "express";
+import { Error, isValidObjectId } from "mongoose";
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)